From b420e279691f8ac865ccdbccb377024a55debd41 Mon Sep 17 00:00:00 2001
From: Sebastian Andrzej Siewior <bigeasy@linutronix.de>
Date: Fri, 24 Feb 2012 18:16:07 +0100
Subject: [PATCH] build-sys: add a crosscompile path for
 scanf_cv_alloc_modifier

This allows to building the package cross again. Since the return value
is not used I guess there is little loss.

Signed-off-by: Sebastian Andrzej Siewior <bigeasy@linutronix.de>
---
 configure.ac |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/configure.ac b/configure.ac
index eab9f85..966e9da 100644
--- a/configure.ac
+++ b/configure.ac
@@ -383,6 +383,8 @@ AC_CACHE_VAL([scanf_cv_alloc_modifier],
       [scanf_cv_alloc_modifier=as],
       [scanf_cv_alloc_modifier=no]
     )
+    ,
+    [scanf_cv_alloc_modifier=no]
   )
 )
 
-- 
1.7.5.4

